sith.py

```diff
@@ -95,6 +94,7 @@ def from_cache(cls, record):
             args = json.load(f)
         return cls(*args)
 
+    # Changing this? Also update the module docstring above.
```

Comment:
Why this comment?

Reply:
As a prompt to keep the docstring and the actual list of valid options in sync (hopefully preventing the need for PRs like this one in the future).

The alternative might be to move to using `docopt`s `(a|b|c)` spelling and remove the checking in this class, but the comment seemed less intrusive.